test/comparison-tests/customTransformer/expectedOutput-2.3/output.txt

```diff
@@ -0,0 +1,5 @@
+    Asset     Size  Chunks             Chunk Names
+bundle.js  2.92 kB       0  [emitted]  main
+chunk    {0} bundle.js (main) 193 bytes [entry] [rendered]
+    [0] ./.test/customTransformer/SUBMODULE/SUBMODULE.ts 78 bytes {0} [built]
```

Comment:
transformer uppercased the module import as well :) it was unintentional, but ok ))